Bottom: b59ef3522bfc1e42b140680a2915de9abb0da877
Top:    b57b4d9bba3bcff69b6642261aa1f69e9b278c0b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:19:22 +0000

Refresh of disable-polygon-heritage

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5dfa616..d6f8ead 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2415,8 +2415,6 @@ add_dummy_descriptors_at_point (Vector point, char poly, CVCList * list, POLYARE
   POLYAREA *b = bfst;
   PLINE *cur;
 
-  pcb_fprintf (stderr, "LOOKING FOR VERTICES AT POINT (%$mn, %$mn)\n", point[0], point[1]);
-
   assert (b != NULL);
   do
     {
